trikot-kword/sample/common/TRIKOT_FRAMEWORK_NAME.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |spec|
     spec.name                     = 'TRIKOT_FRAMEWORK_NAME'
-    spec.version                  = '3.3.4'
+    spec.version                  = '3.3.5'
```

Comment:
I think that this number will be incremented automatically by the release script? @npresseault

Reply:
It is, the file is automatically generated. I wonder if we should keep it under source control.